bin/post_process_sce.R

```diff
@@ -133,6 +125,10 @@ if (alt_exp %in% altExpNames(sce)) {
     }
   }
 
+  # make extra sure there are no NAs in `adt_scpca_filter`
+  if (sum(is.na(sce$adt_scpca_filter)) != 0) {
```

Comment:
another little simplification
```suggestion
  if (any(is.na(sce$adt_scpca_filter))) {
```